### PR TITLE
Migrate Vagrant provisioner to Shell

### PR DESCRIPTION
**Pull request by danny-cheung**

### CODE REVIEW

File: bash/provision.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+sudo apt-get update
+
+ssh-keygen -b 4096 -f ~/.ssh/id_rsa -N '' -q
+cat ~/.ssh/id_rsa.pub >> ~/.ssh/authorized_keys
+cat ~/.ssh/id_rsa.pub >> ~vagrant/.ssh/authorized_keys
+
+echo 'Port 2222' | sudo tee -a  /etc/ssh/sshd_config
+sudo service ssh reload
+
+sudo apt-get install -y git
```
Comment (siso) [inline bash/provision.sh]: Debian package manager should be `aptitude`

File: Vagrantfile [hunk 2]
```diff
@@ -10,12 +10,6 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
...
-  config.vm.provision 'ansible' do |ansible|
-    ansible.sudo = true
-    ansible.inventory_path = 'ansible/inventory/vagrant'
-    ansible.playbook = 'ansible/main.yml'
-    ansible.verbose = 'v'
-    ansible.limit = 'development'
-  end
-
+  config.vm.provision 'shell',
+    path: "bash/provision.sh"
```
Comment (siso) [inline Vagrantfile:14]: Default provisioner should be _Ansible_, as it is far more portable than _BASH_

**Comment by siso** [comment]
Hey @dann7387, thanks for this.
Could you please rename `bash/provision.sh` to `bin/debian-prerequisites.sh`, and provide instructions on `README.md`?

**Comment by danny-cheung** [comment]
@siso The changes have been made.

**Comment by siso** [comment]
Hey @dann7387, closing this PR as Packer provided an easier way to run `chef-takeaway` with `vagrant`

**Pull request closed without merging**